src/main/scala/org/broadinstitute/dsde/rawls/expressions/ExpressionParsing.scala

```diff
@@ -87,6 +91,10 @@ class ExpressionParser extends JavaTokenParsers {
     rootDot ~ rep(entityRefDot) ~ entityRef ^^ {
       case root ~ Nil ~ last => PipelineQuery(List(root) :+ last, outputEntityResult)
       case root ~ ref ~ last => PipelineQuery(List(root) ++ ref :+ last, outputEntityResult)
+    } |
+    workspaceDot ~ rep(entityRefDot) ~ entityRef ^^ {
```

Comment:
I think you're correct in omitting a workspace-only expression here, since it's not valid to run methods on workspaces. That being the case, `private def workspace: Parser[PipeFunc]` isn't necessary.